news/create-newsfile.py

```diff
@@ -106,6 +119,26 @@ def create_standard_blocks():
 
 
 def create_credits_block():
+    def join_with_length_limit(contributors):
+        limit = 70
+        lines = ['']
+
+        for contributor in contributors:
```

Comment:
Do we want to concatenate contributors using `", "`, then wrap at 70 characters? Or there is something more to it?

Could this work?
```
import textwrap
textwrap.fill(", ".join(contributors), 70)
```

Reply:
`textwrap.fill()` It can start a new line at the middle of names, like `Attila Szakacs`. I didn't find an option to modify the delimiter, but I could workaround it with swapping `" "` to `"\0"` before `fill()` and swap back after it.